src/main/java/seedu/address/model/person/Address.java

```diff
@@ -10,6 +10,8 @@
 public class Address {
 
     public static final String MESSAGE_CONSTRAINTS = "Addresses can take any values, and it should not be blank";
+    public static final String MESSAGE_CONSTRAINTS_PROFESSOR =
+            "Please input the professor's office address. Use 'a/ADDRESS'.";
```

Comment:
Use the prefix constant instead of hard code `a/`. (For consistently with the existing code)